Corona-Warn-App/src/main/res/layout/include_submission_done_content_no_consent.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+
+    <import type="de.rki.coronawarnapp.risk.RiskLevelConstants" />
+
+        <variable
+            name="illustrationDescription"
```

Comment:
What is this variable for?

Reply:
I believe this for accessibilty reason (Talkback)

@d4rken  please correct me if I'm wrong

Reply:
I cannot see that you are using this anywhere. Same for the import above.

Reply:
I beileve this has been resolved

Reply:
No, as it is not used anywhere, you should remove it